components/PlantList.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
this looks great! sorry for the change, but is it possible to create the `seasonal-planting-guide` page (route: `app/seasonal-planting-guide`) that uses the `PlantList` component?